docs/swagger-ui/config.json

```diff
@@ -0,0 +1,129 @@
+{
```

Comment:
[question] how can we verify that this file behaves as expected?

When I put https://raw.githubusercontent.com/celestiaorg/celestia-app/9a16d47ec3b27fcf6879ec2d1b83155f56ad7c0f/docs/swagger-ui/config.json into https://petstore.swagger.io/ I observe 

<img width="1463" alt="Screenshot 2023-11-08 at 9 55 52 PM" src="https://github.com/celestiaorg/celestia-app/assets/3699047/3e41ffe5-c0d9-4904-b9d9-dd248caa9526">

is there an alternative Swagger UI we can use to verify this file?

Reply:
you need to generate the swagger file
here is an example: https://github.com/gregnuj/celestia-app/blob/main/docs/swagger-ui/swagger.yaml

Reply:
Should the swagger file be checked into version control? I see:
1. https://github.com/osmosis-labs/osmosis/blob/main/client/docs/static/swagger/swagger.yaml
2. https://github.com/cosmos/gaia/blob/main/client/docs/swagger-ui/swagger.yaml

so I assume yes